test_communication/CMakeLists.txt

```diff
@@ -127,6 +127,32 @@ if(BUILD_TESTING)
     )
   endfunction()
 
+  function(custom_security_test_c target)
+    set(SKIP_TEST "")
+    ament_add_gtest(
+      "${target}${target_suffix}" ${ARGN}
+      TIMEOUT 90
+      ${SKIP_TEST}
```

Comment:
Yeah I kept it originally because I expected the check for rmw implementation to be performed within this function rather than outside. Now that we don't expand templates or register tests for rmw implementation other than Fast-RTPS it can be removed